src/conky.cc

```diff
@@ -495,6 +498,19 @@ int calc_text_width(const char *s) {
 #endif /* BUILD_X11 */
 }
 
+#ifdef BUILD_X11
+#ifdef BUILD_XFT
+int xft_dpi_scale(int value) {
```

Comment:
Change the inner body to something like this to avoid the macro:

```c
#if defined(BUILD_X11) && defined(BUILD_XFT)
    if (use_xft.get(*state)) {
        return (value * xft_dpi + (value>0?48:-48)) / 96;
    } else {
        return value;
    }
#else /* defined(BUILD_X11) && defined(BUILD_XFT) */
    return value;
#endif /* defined(BUILD_X11) && defined(BUILD_XFT) */
```